### PR TITLE
Switch banner to a Firefox download CTA in non-Firefox browsers

### LINKED ISSUE #28: Provide Firefox Download link if user is in a different browser
If a user visits TRF from a non-firefox browser, we should prompt them to download firefox:

Spec is here: https://mozilla.github.io/testpilot-assets/ThemesRFun/01_Measurements/ThemesRFun/#artboard2

Note: do not implement the dismiss button

URL should be something like this with the final name slotted in:
```
https://www.mozilla.org/firefox/new/?utm_campaign=${tbd}-acquisition&utm_medium=referral&utm_source=${tbd}.firefox.com
```

### PR DESCRIPTION
**Pull request by lmorchard**
Fixes #28

### CODE REVIEW

File: src/web/lib/components/App/index.js [hunk 1]
```diff
@@ -26,6 +26,7 @@ const mapStateToProps = state => {
     {}
   );
   return {
+    isFirefox: navigator.userAgent.toLowerCase().includes('firefox/'),
```
Comment (chuckharmston) [inline src/web/lib/components/App/index.js]: Do we want to differentiate Fennec and iOS here?

File: src/web/lib/components/Banner/index.js [hunk 2]
```diff
@@ -1,17 +1,29 @@
...
-    <div className="banner__content">
-      <span>Put the 🔥 in Firefox</span>
-      <a href={addonUrl} className="banner__button">
-        Install Themer
-      </a>
-    </div>
-    {!bottom && <div className="banner__spacer"/>}
-  </div>;
+export const Banner = ({ isFirefox, addonUrl, bottom = false }) => (
+  <div className={classNames('banner', { 'banner--bottom': bottom })}>
```
Comment (chuckharmston) [inline src/web/lib/components/Banner/index.js]: We have `banner--bottom` and `banner__content` here. Could we standardize on a delimiter?

File: src/web/lib/components/App/index.js (hunk 1 above)
Comment (lmorchard) [reply src/web/lib/components/App/index.js]: Hmm, probably - though now that you mention it, I don't think we want to just display "Download firefox" for those browsers. Maybe we need a third message that pushes folks to desktop Firefox in the mobile case.

File: src/web/lib/components/Banner/index.js (hunk 2 above)
Comment (lmorchard) [reply src/web/lib/components/Banner/index.js]: Sure? Probably a @johngruen question, since this comes from an earlier commit and I just reformatted here

**Comment by lmorchard** [comment]
Borrowed some user agent stuff from Test Pilot, hiding the banner altogether for mobile. We can circle back with a more specific CTA if necessary - maybe as part of #45

**Pull request merged**